Gemfile

```diff
@@ -50,6 +50,8 @@ gem 'bootsnap', require: false
 
 gem 'rspec-rails'
```

Comment:
- [**Optional**] Kindly place this gem under the development and test group. That's

```.ruby
 group :development, :test do
         gem 'rspec-rails'
           ...
```

> **_Explanation:_**
   > It needs to be in the :development group to expose generators and rake tasks without having to type RAILS_ENV=test.